src/js/events/MarathonActions.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
 
   createGroup: function (data) {
     RequestUtil.json({
-      url: `${Config.rootUrl}/marathon/v2/groups`,
+      url: `${Config.rootUrl}/service/marathon/v2/groups`,
```

Comment:
do we wanna pull out `/service/marathon/v2` into a constants file so we don't have to do this whole dance again?